yb-voyager/src/query/queryissue/issues_dml.go

```diff
@@ -76,8 +81,8 @@ func NewRegexFunctionsIssue(objectType string, objectName string, sqlStatement s
 
 var aggregateFunctionIssue = issue.Issue{
 	Type:            AGGREGATE_FUNCTION,
-	TypeName:        AGGREGATION_FUNCTIONS_NAME,
-	TypeDescription: "any_value, range_agg and range_intersect_agg functions not supported yet in YugabyteDB",
+	Name:        AGGREGATION_FUNCTIONS_NAME,
```

Comment:
missed adding impact for a few of these?

Reply:
yeah, i think these came in after merging the main branch. Will do add.

Reply:
Although i do remember adding for these during the rebase, but seems like they got removed during rebase of further commits... not sure why.